includes/tools.h

```diff
@@ -49,6 +49,8 @@ typedef enum {
     TIMEOUT_ERROR = 3,
     ARGUMENT_ERROR = 4,
     NO_TIMERS_ERROR = 6,
+    INCORRECT_PRESSURE_ERROR = 7,
+    NO_CONFIG_ERROR = 8,
```

Comment:
Czym to się różni od ARGUMENT_ERROR?

Reply:
oba moga być zwrócone przy odczycie danych i ostrzegają że dane z odczytu są nie poprawne bo np. odczyt ciśnienia jest wyłaczony (tak jest taka opcja i ona jest domyślna nawet)
INCORRECT_PRESSURE_ERROR - oznacza że odczyt ciśnienia jest wyłaczony, ale temperatura została włączona (defaultowo też jest wyłaczona)
NO_CONFIG_ERROR - oznacza że zarówno odczyt temperatuy jak i ciśnienia jest wyłaczony co najprawdopodobniej oznacza że ktoś nie wywołał set_config a więc odczytywanie danych jest bez sensu